package.json

```diff
@@ -29,6 +29,7 @@
   "devDependencies": {
     "@antora/cli": "^3.0.1",
     "@antora/site-generator-default": "^3.0.1",
+    "@tinymce/antora-extension-livedemos": "^0.0.1-rc.20220624042543634.shaf1b28a3",
     "@tinymce/moxiedoc": "^0.2.0-rc.20220310010105103.sha4bab33b",
     "dotenv": "^16.0.0",
     "ecstatic": "^4.1.4",
```

Comment:
The `liquidjs` dependency needs removing below as well.

- [ ] Remove `liquidjs` dev dependency

Reply:
`liquidjs` is already removed

Reply:
You removed it after I'd left this comment, but yes it's been removed now.